### DISCUSSION TITLE
Make shield's ConditionYellow/ConditionRed customizable

### DISCUSSION THREAD

**Discussion by mevitar** [Phobos-developers/Phobos#1212] [Ideas and feedback]
### Description

Shields can have their pips customizable, so it would make sense to give them their own ConditionYellow/ConditionRed setting, separate from the global one used by regular health.

### INI code

```ini
[AudioVisual]
Shield.ConditionYellow=
Shield.ConditionRed=

(defaults to [AudioVisual]->ConditionYellow/ConditionRed )
```

### Additional context

While each shield can have its own pips customizable, it's probably unnecessary to make every shield type have its own ConditionYellow/ConditionRed setting.

### Checklist

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this topic.
- [X] I used the GitHub search and read the topic list to find a similar topic and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

**Comment by Starkku** [comment]
Donezo.

**Comment by mevitar** [comment]
Tested, everything seems to work fine.
